config.example.yml

```diff
@@ -11,8 +11,10 @@ cpus: 2
 # - shortname: The site's shortname.
```

Comment:
Probably, yes. I hesitated to do so, though, because most users probably won't want/need it. Also, we'd need a separate example config file, as shared_folders and synced_folder can't exist in the same config, and I wanted to make it copy/paste friendly.